Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.yml

```diff
@@ -1088,6 +1087,9 @@ script:
     - contextPath: TenableIO.Vulnerability.indexed
       description: The date and time (in Unix time) when the vulnerability was indexed into Tenable.io.
       type: Date
+  - arguments: []
+    description: List the Tenable.io Cloud Scanners
+    name: tenable-io-list-scanners
```

Comment:
Please add an `outputs` key with a list of the context keys this command adds. You can see other commands as a refernce.